services/notify/src/api.js

```diff
@@ -173,7 +172,7 @@ builder.declare({
 
   await req.authorize(req.body);
   try {
-    await this.DenylistedNotification.create(address);
+    await this.db.fns.add_denylist_address(address);
   } catch (e) {
     if (e.name !== 'EntityAlreadyExistsError') {
```

Comment:
Now that we no longer are using tc-lib-entities, we should change the errors to be compatible with postgres. For example, this line should probably be:

```
const { UNIQUE_VIOLATION } = require('taskcluster-lib-postgres');

// ...

if (e.code !== UNIQUE_VIOLATION) {
```

A test that asserts this specific error is thrown when the row already exists might be good as well.

Reply:
Ah, nice spot, thanks!

Reply:
This code disappeared in the end due to https://github.com/taskcluster/taskcluster/pull/3103#discussion_r447998772